Gemfile

```
@@ -0,0 +1,13 @@
+source "https://rubygems.org"
```

rubocop doesn't like double-quoted strings when they are not used for interpolation / special symbols.

```suggestion
source 'https://rubygems.org'
```